vita3k/renderer/src/texture_cache.cpp

```diff
@@ -505,6 +505,7 @@ void upload_bound_texture(const TextureCacheState &cache, const SceGxmTexture &g
         case SCE_GXM_TEXTURE_CUBE_ARBITRARY:
             width = next_power_of_two(width);
             height = next_power_of_two(height);
+            break;
```

Comment:
This break is not needed and will break the graphics of some games (if the texture type is arbitrary adjust the width and the height, then do like the others).

Reply:
@Macdu, fix it https://github.com/Vita3K/Vita3K/pull/2097/commits/2fe6d689e144335f27e6fa669216875a388de3a5